app/src/services/auth-service.ts

```diff
@@ -0,0 +1,30 @@
+import {Injectable, provide} from 'angular2/core';
+
+export class AuthService {
+
+  constructor() {}
+
+  login(user: string, password: string): boolean {
+    if (user === 'user' && password === 'pass') {
+      localStorage.setItem('username', user);
+      return true;
+    }
+    return false;
+  }
+
+  logout(): any {
+    localStorage.removeItem('username');
+  }
+
+  getUser(): any {
+    return localStorage.getItem('username');
+  }
+
+  isLogged(): boolean {
```

Comment:
I'd rename this to isLoggedIn to be a little more explicit - would need to update it in the other components/template also. Minor nitpick, but otherwise looking ok.